app/controllers/concerns/sessions_controller_override.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SessionsControllerOverride
+  def create
+    super do |user|
+      verify_users_after_sign_in(user)
+    end
+  end
```

Comment:
```suggestion
extend ActiveSupport::Concern

  included do
    after_action :csv_auto_verify, only: :create
  end
  
  def csv_auto_verify
  ...
  end
```

I'm not sure this is not overriding the custom decidim implementation. Let's better create a concern and add a hook AFTER the create method

Reply:
Done